docs/guides/dao-members/staking.md

```diff
@@ -73,7 +73,7 @@ DAO pool (as a percentage of the DAO pool). Watch the
 [Stake and Earn](/guides/dao-members/videos.md#stake-and-earn) video.
 
 1. Click the **_Stake_** button.
-   > The tokens available to stake depends on the number deposited and are
+   > The tokens available to stake depend on the number deposited and are
```

Comment:
I'd prefer this be 'The number of tokens available to stake depends'

Reply:
Hey sure, just updated @dcroote